test/files/neg/t0418.check

```diff
@@ -1,4 +0,0 @@
-t0418.scala:2: error: not found: value Foo12340771
-  null match { case Foo12340771.Bar(x) => x }
-                    ^
-one error found
```

Comment:
To apologize for slowing down partest, I deleted this test, which was a duplicate of `test/files/neg/t418.scala`. I hope everyone can live with the added uncertainty.

Reply:
They should give out little medals like they do on stack overflow. Or "merit badges".